src/MuTalk-Tests/MutationTestingTimeBudgetTest.class.st

```diff
@@ -66,8 +60,8 @@ MutationTestingTimeBudgetTest >> testWithNoTimeForMutantsRunsNoMutants [
 	duration := self fixedAnalysisTime / 4.
 	analysisTime := [ result := self runAnalysisFor: duration ] timeToRun.
 
-	"We always treat at least one mutant"
-	self assert: result numberOfEvaluatedMutants equals: 1
+	"We never treat mutants"
```

Comment:
Now reading this code one week later I tell myself we should better explain in the comment.
```suggestion
	"We never treat mutants because the budget was exceeded running the initial test run, coverage analysis, etc"
```